python/ray/state.py

```diff
@@ -183,7 +183,7 @@ def object_table(self, object_ref=None):
             for i in range(len(object_table)):
                 object_location_info = gcs_utils.ObjectLocationInfo.FromString(
                     object_table[i])
-                results[binary_to_hex(object_location_info.object_ref)] = \
+                results[binary_to_hex(object_location_info.object_id)] = \
```

Comment:
I think these should actually get changed to `object_ref`

Reply:
In the protobuf, it seems to be just `bytes` type, which looks like just the ID? Isn't `objectRef` indicating the whole object? (I might misunderstand this)

Reply:
Oh, right, this seems fine then.